tests/modules/get_professors_by_class/app/get_professors_by_class_controller.test.ts

```diff
@@ -0,0 +1,89 @@
+import { it, expect, describe } from 'vitest';
+import { GetProfessorsByClassUsecase } from '../../../../src/modules/get_professors_by_class/app/get_professors_by_class_usecase';
+import { GetProfessorsByClassController } from '../../../../src/modules/get_professors_by_class/app/get_professors_by_class_controller';
+import { HttpRequest } from '../../../../src/shared/helpers/external_interfaces/http_models';
+import { ScheduleRepositoryMock } from '../../../../src/shared/infra/repositories/schedule_repository_mock';
+
+describe('Assert GetProfessorsByClassController is correct at all', () => {
+  it('should return a list of professors for a valid classId', async () => {
```

Comment:
Create a test with the classId being a string, but not a UUID (you will see that the controller does not validate this...)